rust/agents/scraper/migration/bin/common.rs

```diff
@@ -16,6 +18,8 @@ pub async fn init() -> Result<DatabaseConnection, DbErr> {
         .init();
 
     let url = url();
+    let mut options: ConnectOptions = url.clone().into();
+    options.connect_timeout(Duration::from_secs(CONNECT_TIMEOUT));
```

Comment:
I don't disagree w/ this but curious what motivated this

Reply:
Random failures in the pipeline, and this was a quickfix I found in https://github.com/SeaQL/sea-orm/issues/742#issuecomment-1129875130